db/migrate/20150511140404_create_popup_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePopupNotifications < ActiveRecord::Migration
+  def change
+    create_table :popup_notifications do |t|
+      t.string :position
+      t.string :title
+      t.text :content
+      t.string :button_text
+      t.string :image_url
+      t.string :link
+      t.boolean :sharable
```

Comment:
It is used to determine if this popup should provide a button(or something else) which could handle sharing to FB(or something else)